modules/ipi-install-out-of-band-management.adoc

```diff
@@ -6,8 +6,18 @@
 [id="out-of-band-management_{context}"]
 = Out-of-band management
 
-Nodes will typically have an additional NIC used by the baseboard management controllers (BMCs). These BMCs must be accessible from the provisioner node.
+Nodes typically have an additional NIC used by the baseboard management controllers (BMCs). These BMCs must be accessible from the provisioner node.
 
 Each node must be accessible via out-of-band management. When using an out-of-band management network, the provisioner node requires access to the out-of-band management network for a successful {product-title} installation.
 
-The out-of-band management setup is out of scope for this document. We recommend setting up a separate management network for out-of-band management. However, using the provisioning network or the baremetal network are valid options.
+The out-of-band management setup is out of scope for this document. Using a separate management network for out-of-band management can enhance performance and improve security. However, using the provisioning network or the baremetal network are valid options.
```

Comment:
Kicking this back due to the use of "baremetal." According to our supplementary guide, "baremetal" is an incorrect usage: https://redhat-documentation.github.io/supplementary-style-guide/#bare-metal-n 

I think in this context its a noun, and should be "bare metal."